packages/react/src/internal/FeatureFlags.js

```diff
@@ -142,6 +142,14 @@ export const DDS_FEATURED_LINK =
 export const DDS_CONTENT_GROUP =
   process.env.DDS_CONTENT_GROUP === 'true' || DDS_FLAGS_ALL || false;
 
+/**
+ * This determines if the ContentItem component will be rendered or not
+ *
+ * @type {string | boolean}
+ */
+export const DDS_CONTENT_ITEM =
+  process.env.DDS_CONTENT_ITEM === 'true' || DDS_FLAGS_ALL || false;
+
```

Comment:
@annawen1 we're going straight to prod qa with this one, let's exclude the feature flag.

Reply:
ok, removed!